website/docs/guides/fixtures/books.js

```diff
@@ -0,0 +1,239 @@
+db.books.insertMany([
```

Comment:
Just structural question, how should the files be named? :thinking: 

It seems may files are named like
`website/docs/guides/cts/1-create-ttl-index.json` -> cts
`website/docs/guides/requests/1-create-ttl-index/create-ttl-index.js` -> request
`website/docs/guides/responses/1-create-ttl-index/create-ttl-index.js` -> response

But this file seems different
`website/docs/guides/cts/books.fixture.json` -> cts
`website/docs/guides/fixtures/books.js` -> request
no response used

Reply:
`.json` files contain testsuites with requests and responses;
`.fixture.json` files contain documents that should be inserted to collection for test purposes.